content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -82,6 +82,13 @@ The Condition Resource supports proprietary codes for:
 
 * Condition.severity codes are maintained in [Code Set 12022 Severity](#code-set-12022-severity)
 
+<!-- use html header to avoid showing up in toc -->
```

Comment:
Do we need this comment?

Reply:
Removed [here](https://github.com/cerner/fhir.cerner.com/pull/322/commits/783dce5f317c0ee97b87117e79fb62dcbca23583#diff-6fdcb00b5b1b6edd51b687794bea6b95L85).

Reply:
for future reference, I do prefer to leave that comment in place as it explains why the html headers are required instead of markdown headers